contract/src/test/kotlin/io/provenance/scope/loan/utility/DataConversionExtensionsUnitTest.kt

```diff
@@ -57,6 +61,9 @@ class DataConversionExtensionsUnitTest : WordSpec({
             }
         }
     }
+    "toMISMOLoan" xshould {
+        // TODO: Implement
```

Comment:
Did you intend to fill this in for this PR or a future one? (Just a reminder if wanted it now)

Reply:
A future PR. I was prioritizing test cases for the contract function changes, and the ones I've added in this PR do indirectly call this function such that if there was a problem with it, it would throw an exception not of the type expected by `shouldThrow<>`, so it should be fine to omit for now.